cudapoa/include/claraparabricks/genomeworks/cudapoa/utils.hpp

```diff
@@ -50,6 +57,136 @@ void get_multi_batch_sizes(std::vector<BatchSize>& list_of_batch_sizes,
                            int32_t gap_score                   = -8,
                            int32_t match_score                 = 8);
 
+/// \brief Parses window data file
```

Comment:
I think it would be good to have an example of this data file. Can you add it to the description?
```
<num_sequences_in_group_0>
group0_seq0
group0_seq1
group0_seq2
...
...
<num_sequences_in_group_1>
group1_seq0
group1_seq1
group1_seq2
...
...
```

Reply:
Added. Renamed to windows to keep external interface name consistent.